pages/ListPage.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import CampingCard from "../components/CampingCard";
+import { campingTrips } from "../data/campingTrips";
+import { Container, Card, Header, Footer } from "../styles/styles";
+
+export default function ListPage() {
+  return (
+    <>
+      <Header>
+        <h1>Organize your Trips</h1>
+      </Header>
+      <Container>
+        {campingTrips.map((trip) => (
+          <Card key={trip.id}>
+            <CampingCard trip={trip} />
+          </Card>
+        ))}
+      </Container>
+      <Footer>
+        <p>© 2023 Your App</p>
+      </Footer>
```

Comment:
this makes more sense in your index.js, as it will stay the same no matter what component you are rendering in the main-part of your app?